src/components/Page/Page.jsx

```diff
@@ -12,18 +12,16 @@ import '../Sponsors/Sponsors.scss';
 import '../Gitter/Gitter.scss';
 
 const Page = ({ page, section }) => {
-  const {
-    file: {
-      attributes: {
-        anchors,
-        contributors,
-        title,
-        related,
-      },
-      body
-    },
-    url
-  } = page;
+  let { contributors, title, related } = page.file.attributes;
+  let pages = (
+    section.pages()
+      .sort((a, b) => a.file.attributes.sort - b.file.attributes.sort)
```

Comment:
About this sort, it makes the list different from the current home page.
![sort](https://user-images.githubusercontent.com/12815768/29891348-f3bcc1fe-8df4-11e7-80c4-7e16f7ff8b68.PNG)
Using `transform` at `antwar.config` is working normally.